core/src/main/kotlin/io/saagie/updatarium/model/ChangeSet.kt

```diff
@@ -79,15 +83,16 @@ data class ChangeSet(
      * Check if the change set must be run, either because it is mark as 'force' through its parameters or
      * because it has not been already executed.
      **/
-    private fun mustRun(executionId: String, configuration: UpdatariumConfiguration): Boolean =
+    internal fun mustRun(executionId: String, configuration: UpdatariumConfiguration): Boolean =
         if (force) {
             logger.warn { "$executionId marked as forced. Will be executed" }
             true
         } else {
             when (configuration.persistEngine.findLatestExecutionStatus(executionId)) {
-                NOT_EXECUTED -> true
+                OK, MANUAL_OK -> false
+                EXECUTE -> false
```

Comment:
Why merged **ok** and **manual_ok** but not **execute** ?

Reply:
OK and MANUAL_OK are the same in this case.
EXECUTE is another status.

I will be possible to merge it with the OKs but I think it's better to dissociate status familly with their meaning and not their return value